Intersect.Client/Entities/Player.cs

```diff
@@ -310,17 +310,15 @@ public override void Load(EntityPacket packet)
             Guild = pkt.Guild;
             Rank = pkt.GuildRank;
 
-            var playerPacket = (PlayerEntityPacket)packet;
-
-            if (playerPacket.Equipment != null)
+            if (pkt.Equipment != null)
```

Comment:
Rather than using the pkt rename, change `pkt` (which is a bad variable name) to `playerPacket` -- this will change L306-L311 and leave L313+ untouched.

Reply:
Changed :)